gnovm/stdlibs/std/native.go

```diff
@@ -32,6 +32,37 @@ func GetHeight(m *gno.Machine) int64 {
 	return m.Context.(ExecContext).Height
 }
 
+// getPrevFunctionNameFromTarget returns the last called function name (identifier) from the call stack.
+func getPrevFunctionNameFromTarget(m *gno.Machine, targetFunc string) string {
+	targetIndex := findTargetFuncIndex(m, targetFunc)
+	if targetIndex == -1 {
+		return ""
+	}
+	return findPreviousFuncName(m, targetIndex)
+}
+
+// findTargetFuncIndex finds and returns the index of the target function in the call stack.
+func findTargetFuncIndex(m *gno.Machine, targetFunc string) int {
+	for i := len(m.Frames) - 1; i >= 0; i-- {
+		currFunc := m.Frames[i].Func
+		if currFunc != nil && currFunc.Name == gno.Name(targetFunc) {
+			return i
+		}
+	}
+	return -1
+}
+
+// findPreviousFuncName returns the function name before the given index in the call stack.
+func findPreviousFuncName(m *gno.Machine, targetIndex int) string {
```

Comment:
```suggestion
func findPrevFuncName(m *gno.Machine, targetIndex int) string {
```

for consistency.

Reply:
@notJoon 
This method was still not renamed 🙏

Reply:
@zivkovicmilos I just committed it with all the changes

Reply:
https://github.com/gnolang/gno/pull/1653/commits/317108ae1f0384666ecd592ca02d982ccb689865